.opencode/command/spellcheck.md

```
@@ -2,5 +2,5 @@
 description: Spellcheck all markdown file changes
 ---
 
-Look at all the unstaged changes to markdown (.md, .mdx) files,
-pull out the lines that have changed, and check for spelling and grammar errors.
+Look at all the unstaged changes to markdown (.md, .mdx) files, pull out the
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)